src/CarbonAware.DataSources/CarbonAware.DataSources.Json/src/CarbonAware.DataSources.Json.csproj

```diff
@@ -10,4 +10,11 @@
     <Nullable>enable</Nullable>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
FYI @juzuluag this section puts the `data-files` in the right place for any project that includes this `.csproj` as a dependency, so I cleaned up the references in the WebApi and CLI `.csproj` files.  I was going to need to add those same lines to the `[CarbonAware.CLI.IntegrationTests.csproj](https://github.com/microsoft/carbon-aware-sdk/pull/144/files#diff-cfc48b341537576cbe42c45ba406974796e1495dbbba6bd94ea962d1a54f0f18)` as well to get the build to work and I felt like there had to be an easier way.

Reply:
Interesting. I just tried and it gets copied on all the projects eventually.